app/AccountancyModule/Components/Participants/templates/PersonPicker.latte

```diff
@@ -1,50 +1,77 @@
     {snippet persons}
         <h3>Osoby v jednotce</h3>
-        <div class="well well-sm">
-            <div class="btn-group">
-                <a class="btn btn-primary btn-sm dropdown-toggle" data-toggle="dropdown" href="#" rel="tooltip" data-original-title="Jednotka, ze které se zobrazí seznam členů">{$unit->getDisplayName()|truncate:20, ""} <span class="caret"></span></a>
-                <ul class="dropdown-menu">
-                    <li n:if="$parentUnit !== null"><a n:href="this unitId => $parentUnit->getId()">{$parentUnit->getDisplayName()}</a></li>
-                    <li>
-                        <a class="active" style="font-weight: bold" href="#">&minus;&gt;{$unit->getDisplayName()}</a>
-                    </li>
-                    <li n:foreach="$subunits as $subunit">
-                        <a n:href="this unitId => $subunit->getId()">&minus;&minus;&gt; {$subunit->getDisplayName()}</a>
-                    </li>
-                </ul>
+        <div class="d-flex">
+            <div class="btn-group mr-2">
+                <a class="btn btn-light btn dropdown-toggle d-inline-flex align-items-center" data-toggle="dropdown" href="#" rel="tooltip" title="Jednotka, ze které se zobrazí seznam členů">
+                    {$unit->getDisplayName()|truncate:20, ""}
+                    <small class="text-muted ml-1 mr-1">
+                        {if $directMemberOnly}bez podjednotek{else}+ podjednotky{/if}
```

Comment:
tohle hodně prodlužuje to tlačítko, nenapadá tě nějak jinak to udělat? kdyz dělame truncate na název jednotky, abychom za to daly takovy text mi prijde hloupý

Reply:
Souhlasím, bohužel mě zatím nenapadá, jak to jinak pořešit. Ještě na to kouknu.

Reply:
Sice se mi tam nelíbí ta mezera, ale z pohledu toho místa je to tak lepší:
![image](https://user-images.githubusercontent.com/5658260/79769208-1c034f00-832c-11ea-80af-b347ac2defce.png)
![image](https://user-images.githubusercontent.com/5658260/79769237-21609980-832c-11ea-83d7-1ab2b23e1c79.png)

Co myslíš @sinacek ?

Reply:
je to trochu nesystémový prvek, ale lepší to je